readme.md

```diff
@@ -24,7 +24,7 @@
 
 	   vagrant up
 
-	which will be available at [http://blockextend.local](http://blockextend.local) after provisioning (username: `admin`, password: `password`).
+	which will be available at [blockextend.local](http://blockextend.local) after provisioning.
```

Comment:
Shouldn't we keep the credentials in this string?

Reply:
Yes, absolutely. I removed those because the automatic WP setup during `docker-compose up` didn't work because the DB container still wasn't finished when the WP CLI container `command` would run trying to install WP. I moved the install script to `Vagrantfile` and that works great.